src/Microsoft.Data.SqlClient/tests/ManualTests/AlwaysEncrypted/CspProviderExt.cs

```diff
@@ -194,10 +187,6 @@ public void TestEncryptDecryptWithCSP(string connectionString)
                         }
                     }
                 }
-                catch (Exception e)
-                {
-                    Console.WriteLine(@"Exception: {0}", e.Message);
-                }
                 finally
                 {
```

Comment:
NIT: can we keep the catch block without catching the exception. Those messages are helpful when something happens.

Reply:
This test had been failing the entire time but the exception was being swallowed by the catch. Why not remove it and let the test fail? The underlying exception is printed out either way.